assets/booking_index.js

```diff
@@ -20,15 +21,15 @@ document.addEventListener('DOMContentLoaded', function() {
     let calendarEl = document.getElementById('calendar');
 
     let calendar = new Calendar(calendarEl, {
-        plugins: [ timeGridPlugin, dayGridPlugin,interactionPlugin],
+        plugins: [ timeGridPlugin, dayGridPlugin, interactionPlugin, listPlugin],
         locale:frLocale,
         timeZone: 'Europe/Paris',
         selectable: true,
         allDaySlot:false,
         headerToolbar: {
             start:'prev,next today',
             center: 'title',
-            end: 'dayGridMonth,timeGridDay',
+            end: 'dayGridMonth,timeGridDay,listYear',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/LeonardDeVanGogh/VosgesActivies/pullRequest?prid=7268173)
```suggestion
            end: "dayGridMonth,timeGridDay,listYear",
```